README.md

```diff
@@ -11,15 +11,16 @@
   <a href="https://circleci.com/gh/facebook/docusaurus"><img src="https://img.shields.io/circleci/build/github/facebook/docusaurus.svg" alt="CircleCI status"></a>
   <a href="CONTRIBUTING.md#pull-requests"><img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs Welcome"></a>
   <a href="https://discord.gg/docusaurus"><img src="https://img.shields.io/discord/102860784329052160.svg" align="right" alt="Discord Chat" /></a>
-  <a href="https://github.com/prettier/prettier"><img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg"></a>
+  <a href= "https://github.com/prettier/prettier"><img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg"></a>
+  <a href="#license"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
```

Comment:
What is this badge for?

Reply:
Hi, I made changes to the code of coduct earlier. However, while reviewing, I was asked not to change the code of conduct(which I wasn't aware about) so the changes made to the code of conduct have been reverted. 
Secondly, this shows the license of the product.